schwifty/bank_registry/manual_at.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "country_code": "AT",
+    "bic": "BUNDATWWXXX",
+    "bank_code": "01000",
+    "name": "Republik Österreich",
+    "short_name": "Republik Österreich",
+    "primary": true
+  }
+]
```

Comment:
This bank is already part of the automatically generated registry for Austria. The difference I can see is that the manual curated entry has a leading `"0"` for the domestic bank code. Did you experience any issues in that regard so that it was necessary for you to add a manual overwrite?
https://github.com/mdomke/schwifty/blob/4504b41ca6153103c3fafdfc3392f42f9f8ca463/schwifty/bank_registry/generated_at.json#L6962-L6969

Reply:
Hi, thank you for your comments
Here is a test screenshoot. It seems that it doesn't get "bank_code": "1000" in generated_at.json
So I create a new data file to include "bank_code": "01000"
<img width="500" alt="picture01" src="https://user-images.githubusercontent.com/3207256/132470784-676f0392-aad7-4ff8-aaad-6876544524ee.png">

Reply:
> Hi, thank you for your comments
> Here is a test screenshoot. It seems that it doesn't get "bank_code": "1000" in generated_at.json
> So I create a new data file to include "bank_code": "01000"
> <img alt="picture01" width="500" src="https://user-images.githubusercontent.com/3207256/132470784-676f0392-aad7-4ff8-aaad-6876544524ee.png">

here is the testing IBAN
AT930100000000123145